thoth/user_api/api_v1.py

```diff
@@ -291,20 +291,19 @@ def list_thoth_container_images(
         entries.append(item)
 
     prev_page, next_page = _compute_prev_next_page(page, per_page)
+
+    if len(entries) == 0:
+        return (
+            {"error": "No Image found", "parameters": parameters},
+            404,
+        )
+
     return (
         {
             "container_images": entries,
             "parameters": parameters,
         },
         200,
-        {
-            "page": page,
-            "per_page": per_page,
-            "page_count": page_count,
-            "entries_count": entries_count,
-            "next": next_page,
-            "prev": prev_page,
-        },
```

Comment:
The values are needed here, so user can request the result from next page if needed.

Reply:
Yep reverted